src/utils/color-utils.js

```diff
@@ -60,6 +60,25 @@ const colorUtils = {
     const current = getCurrentTheme(Theme);
     return (current && current.properties.fontFamily) || 'inherit';
   },
+  getFadedColor(color) {
+    const percent = -0.15;
+    let f; let R; let B; let G;
+    const t = 0; // percent < 0 ? 0 : 255
+    const p = percent * -1; // percent < 0 ? percent * -1 : percent
+    if (color.length > 7) {
+      f = color.split(',');
+      const rgba = f[0].indexOf('a') !== -1;
```

Comment:
What happens when I have set my color to a color starting with 'a'? (aliceblue, antiquewhite, aquamarine)